aqua/beekeeper-studio/Portfile

```diff
@@ -32,13 +33,27 @@ checksums           ${distname}${extract.suffix} \
                     sha256  093a41aa23e2d4b2259265a0a9fbe3feef8cfd1e4054417f39bedc0f768d5c1b \
                     size    42107147
 
-depends_build       port:yarn
+depends_build       port:go \
+                    port:yarn
 
 use_configure       no
 
 build {
+    set gopath ${workpath}/go
+
+    # Set up all JS dependencies
     system -W ${worksrcpath} "yarn"
-    system -W ${worksrcpath} "CSC_IDENTITY_AUTO_DISCOVERY=false yarn run electron:build --mac dir"
+
+    # Workaround for: https://trac.macports.org/ticket/60555
+    # Build app-builder-bin locally and insert it into node_modules
+    system "GOPATH=${gopath} go get -v github.com/develar/app-builder"
```

Comment:
Why not add `GOPATH=${gopath}` to the `build.env`?

Reply:
Same issue as above.